version.go

```diff
@@ -1,20 +1,12 @@
 package containerd
 
-import "fmt"
+var (
+	Package = "github.com/docker/containerd"
 
-// VersionMajor holds the release major number
-const VersionMajor = 1
+	// Version holds the complete version number.
+	Version = "1.0-dev+unknown"
 
-// VersionMinor holds the release minor number
-const VersionMinor = 0
-
-// VersionPatch holds the release patch number
-const VersionPatch = 0
-
-// Version holds the combination of major minor and patch as a string
-// of format Major.Minor.Patch
-var Version = fmt.Sprintf("%d.%d.%d", VersionMajor, VersionMinor, VersionPatch)
-
-// GitCommit is filled with the Git revision being used to build the
-// program at linking time
-var GitCommit = ""
+	// GitCommit is filled with the Git revision being used to build the
+	// program at linking time
+	GitCommit = ""
```

Comment:
Unless I'm missing something, `GitCommit` isn't actually being filled in or used anywhere?